Makefile

```diff
@@ -460,3 +465,19 @@ else
 	mkdir -p $(dir $@)
 	uuidgen | tr '[:upper:]' '[:lower:]' > $@
 endif
+
+
+###########
+## Tools ##
+###########
+
+# Local Node Scanner
+.PHONY: local-nodescanner
+local-nodescanner:
```

Comment:
is the idea that this should only be used inside a container? Otherwise, Mac users won't be able to use the binary directly :(

Reply:
The idea is to have maximum flexibility. I have added a UBI 9 container target in this PR to make it runnable anywhere, but we could add more, e.g. UBI 8 or even Fedora latest for a bleeding edge RPM version.
The binary can either be used for tests and/or CI, or as a debug entry point for local debugging. That local debugging would not work on bare metal OSX w.r.t. RPM packages, but still might be useful, or could be done in the container.